components/projects/NewProject/NewProjectForm.tsx

```diff
@@ -29,9 +30,32 @@ function NewProjectForm({ employees, customers }: NewProjectFormProps): JSX.Elem
         }
     };
 
-    const handleSumbit = (e: React.FormEvent<HTMLFormElement>) => {
+    const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
         e.preventDefault();
-        console.log(state);
+        // console.log(state);
```

Comment:
Remove all // console.logs, they serve no purpose on the main branch

Reply:
Fixed in df7dea1